From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Alistair Delva <adelva@google.com>
Date: Fri, 14 Feb 2020 13:35:21 -0800
Subject: ANDROID: gki_defconfig: Disable SDCARD_FS

Move SDCARD_FS into the fragments for cuttlefish and db845c as this
feature is still being used by AOSP master. These will be removed later.

Bug: 149410031
Change-Id: I9142338cd0709aa8ec067b9e7d8c22390630de02
Signed-off-by: Alistair Delva <adelva@google.com>
---
 arch/arm64/configs/db845c_gki.fragment | 1 +
 arch/arm64/configs/gki_defconfig       | 1 -
 arch/x86/configs/gki_defconfig         | 1 -
 cuttlefish.fragment                    | 1 +
 4 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/configs/db845c_gki.fragment b/arch/arm64/configs/db845c_gki.fragment
index 5ad26f72e3ad..125453fddfd1 100644
--- a/arch/arm64/configs/db845c_gki.fragment
+++ b/arch/arm64/configs/db845c_gki.fragment
@@ -63,3 +63,4 @@ CONFIG_QCOM_QFPROM=m
 CONFIG_INTERCONNECT_QCOM=y
 CONFIG_INTERCONNECT_QCOM_SDM845=m
 CONFIG_INCREMENTAL_FS=m
+CONFIG_SDCARD_FS=m
diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
index bccd38606d3a..05895995d4f4 100644
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -402,7 +402,6 @@ CONFIG_MSDOS_FS=y
 CONFIG_VFAT_FS=y
 CONFIG_TMPFS=y
 # CONFIG_EFIVAR_FS is not set
-CONFIG_SDCARD_FS=y
 CONFIG_PSTORE=y
 CONFIG_PSTORE_CONSOLE=y
 CONFIG_PSTORE_RAM=y
diff --git a/arch/x86/configs/gki_defconfig b/arch/x86/configs/gki_defconfig
index 954bb841a452..f13dfe016fa7 100644
--- a/arch/x86/configs/gki_defconfig
+++ b/arch/x86/configs/gki_defconfig
@@ -338,7 +338,6 @@ CONFIG_VFAT_FS=y
 CONFIG_TMPFS=y
 CONFIG_TMPFS_POSIX_ACL=y
 # CONFIG_EFIVAR_FS is not set
-CONFIG_SDCARD_FS=y
 CONFIG_PSTORE=y
 CONFIG_PSTORE_CONSOLE=y
 CONFIG_PSTORE_RAM=y
diff --git a/cuttlefish.fragment b/cuttlefish.fragment
index e201d2d23a9a..9a263270fe9e 100644
--- a/cuttlefish.fragment
+++ b/cuttlefish.fragment
@@ -19,3 +19,4 @@ CONFIG_VIRTIO_MMIO_CMDLINE_DEVICES=y
 CONFIG_INCREMENTAL_FS=m
 CONFIG_TEST_STACKINIT=m
 CONFIG_TEST_MEMINIT=m
+CONFIG_SDCARD_FS=m
